website/client/components/tasks/taskModal.vue

```diff
@@ -691,6 +700,7 @@ export default {
         calendar: calendarIcon,
       }),
       requiresApproval: false, // We can't set task.group fields so we use this field to toggle
+      sharedCompletion: 'individualCompletion',
```

Comment:
From the tests, `individualCompletion` is not an option.

Reply:
Can you elaborate? Do you mean I should add a test that the behavior stays the same (i.e., that the master task is never completed) while it's set to `individualCompletion`?